openreview/conference/matching.py

```diff
@@ -251,6 +251,92 @@ def _build_conflicts(self, submissions, user_profiles, get_profile_info):
             return self._build_profile_conflicts(ac_profiles, user_profiles)
         return self._build_note_conflicts(submissions, user_profiles, get_profile_info)
 
+    def append_note_conflicts(self, user_profile, build_conflicts=None):
+        '''
+        Create conflict edges between the given Notes and a single profile
+        '''
+
+        # Re-setup information that would have been initialized in setup()
+        submissions = list(openreview.tools.iterget_notes(
+            self.conference.client,
+            invitation=self.conference.get_blind_submission_id(),
+            details='original'))
+        get_profile_info = openreview.tools.get_neurips_profile_info if build_conflicts == 'neurips' else openreview.tools.get_profile_info
+        self.match_group = openreview.tools.replace_members_with_ids(self.client, self.match_group)
```

Comment:
You don't need to run this. It can be very expensive. I would make sure the profile_id is a valid profile.